### DISCUSSION TITLE
NextJs Example

### DISCUSSION THREAD

**Discussion by esharmony** [TanStack/query#1857] [Q&A]
Whilst looking at the NextJS example I am confused by one thing
In the post list component you have an **isClient** property

```
export const PostList = ({ isClient }) => {
  const [postCount, setPostCount] = useState(10)
  const { data, isLoading, isFetching } = usePosts(postCount, isClient)
```

this isn't being passed from the page, and also does not seem to be used in usePosts to which it is being passed.
Is this not used, or am i missing something here. Many thanks

**Comment by TkDodo** [comment]
I think it's just an error in the example. Feel free to open a PR that removes that prop, as you said, it's unused. A similar question came up before (https://github.com/tannerlinsley/react-query/discussions/1365), and it rightfully confuses people :)

**Reply by esharmony** [reply]
OK, many thanks for letting me know.

**Reply by esharmony** [reply]
> I think it's just an error in the example. Feel free to open a PR that removes that prop, as you said, it's unused. A similar question came up before (#1365), and it rightfully confuses people :)

I don't have permission to create a PR, (I cloned and change and pushed a new branch from git)

**Reply by TkDodo** [reply]
you fork, create a branch and then create a PR from your fork to this repo :)

**Reply by esharmony** [reply]
thanks
